v3.1/glfw/native_darwin.go

```diff
@@ -1,22 +1,29 @@
 package glfw
 
-//#define GLFW_EXPOSE_NATIVE_COCOA
-//#define GLFW_EXPOSE_NATIVE_NSGL
-//#include "glfw/include/GLFW/glfw3.h"
-//#include "glfw/include/GLFW/glfw3native.h"
-import "C"
+/*
+#define GLFW_EXPOSE_NATIVE_COCOA
+#define GLFW_EXPOSE_NATIVE_NSGL
+#include "glfw/include/GLFW/glfw3.h"
+#include "glfw/include/GLFW/glfw3native.h"
 
 // See: https://github.com/go-gl/glfw3/issues/82
```

Comment:
This line should be updated to point to https://github.com/go-gl/glfw/issues/136, or describe the issue.

The funcs below are named "workaround", so it'd be great to mention the context for why a workaround is needed.